src/dp_service.c

```diff
@@ -187,11 +187,11 @@ static void free_interfaces(void)
 	dp_ifaces_free();
 	dp_flow_free();
 	dp_telemetry_free();
+	dp_ports_free();
```

Comment:
Since this was a single commit, it must be helping, but I cannot see why that is. Do you have any details as to why this helps DPDK to clean up properly?

Reply:
In `dp_ports_free()`, the ctx of a graph node is accessed to set a value. this ctx is actually freed together with graph node. Due to the code change within `rte_graph_destroy()`, this invalid access gets exposed.